bin/plotting/pycbc_plot_vt_ratio

```diff
@@ -54,9 +55,13 @@ x_param = r'$' + tuple(ffirst_set['data'].keys())[0].split('\\in')[0].strip('$')
 # read in the positions from the labels
 xpos = np.array([float(l.split('[')[1].split(',')[0]) for l in labels])
 
-# offset different ifars by 1/20th of the distance between one parameter value and the next
-# for the last value use the previous difference
-xpos_add_dx = 0.05 * np.append(np.diff(xpos), np.diff(xpos)[-1])
+# offset different ifars by 1/20th of the mean distance parameter value diffs
+if args.log_x:
+    xpos_logdiffmean = np.diff(np.log(xpos)).mean()
+    xpos_add_dx = 0.05 * np.ones_like(xpos) * xpos_logdiffmean
+else:
+    xpos_diffmean = np.diff(xpos).mean()
+    xpos_add_dx = 0.05 * np.ones_like(xpos) * xpos_diffmean
```

Comment:
I found another case which needs special treatment, namely if there's only one parameter bin so np.diff fails. 
I dealt with this via try / except IndexError and then arbitrarily setting `xpos_add_dx = 0.05`